packages/eslint-plugin-clarity-migration/test/no-clr-alert.spec.ts

```diff
@@ -0,0 +1,233 @@
+import rule from '../src/rules/no-clr-alert';
+import { RuleTester } from './test-helper';
+
+interface InvalidTest {
+  code: string;
+  errors: Array<{
+    messageId: string;
+    line?: number;
+    column?: number;
+  }>;
+  output?: string;
+}
+
+interface Location {
```

Comment:
Is it worth using this type in the `calculateLocation` utility function as something named

```
ElementLocation || TemplateLocation
```
?

Reply:
After looking into this for a while, I've found an appropriate type exported by `@typescript-eslint` and used it instead.